lib/DragDropAction.vala

```diff
@@ -25,8 +25,9 @@ namespace Gala {
     }
 
     public class DragDropAction : Clutter.Action {
-        private static Gee.HashMap<string,Gee.LinkedList<Actor>>? sources = null;
-        private static Gee.HashMap<string,Gee.LinkedList<Actor>>? destinations = null;
+        // DO NOT keep a reference otherwise we get a ref cycle
+        private static Gee.HashMap<string, Gee.LinkedList<unowned Actor>>? sources = null;
+        private static Gee.HashMap<string,Gee.LinkedList<unowned Actor>>? destinations = null;
```

Comment:
A nitpick, but can you remove the space after the comma here please, just to be consistent :)

Reply:
Done :)